runtime/parachains/src/hrmp.rs

```diff
@@ -430,6 +440,22 @@ pub mod pallet {
 			Ok(())
 		}
 
+		/// This cancels a pending open channel request. It can be canceled be either of the sender
+		/// or the recipient for that request. The origin must be either of those.
+		///
+		/// The cancelling happens immediatelly. It is not possible to cancel the request if it is
+		/// already accepted.
```

Comment:
If request is already accepted, why not interpret the `cancel_request` as `close_channel`?

Otherwise most callers of cancel open request will wait until this Errs and then call `close_channel`

Reply:
Correction: we can't issue `close_channel` for a channel that is not yet opened. Nonetheless, I got your idea: why not just permit the cancelling a confirmed open request?

Well, this is just what @shawntabrizi and me decided when we spoke about it. I thought it is a tad bit simpler to  implement this way. Also is not that important in the end, since this is a rather an edge case.

WDYT, @shawntabrizi ?